themes/wilson_theme_starterkit/components/atoms/button/button.stories.js

```diff
@@ -0,0 +1,41 @@
+export default { title: 'Buttons' };
+
+import button from './button.twig';
+import '../../../css/styles.css'
+import drupalAttribute from 'drupal-attribute'
+
+export const default_button = () => (
+    button({
+      attributes: new drupalAttribute(),
+      type: 'primary',
+      items: [
```

Comment:
This could be simplified if the we use my suggestion on the template files.

Reply:
yep simplified now :)